lib/membrane_h264_ffmpeg/encoder.ex

```diff
@@ -134,17 +135,17 @@ defmodule Membrane.H264.FFmpeg.Encoder do
              framerate_denom,
              state.crf
            ) do
-      {{:ok, redemand: :output}, %{state | encoder_ref: encoder_ref}}
+      actions = buffers ++ [redemand: :output]
```

Comment:
I think you should also pass the new caps (e.g. resolution could have changed)

Reply:
currently new caps are created in handle_process - but it is a good idea, I will move them here.